### PR TITLE
Remove child processes from centroid computations

### PR DESCRIPTION
**Pull request by sevko**
The importer currently instantiates a child process per polygon centroid computation, which is insanely expensive and slows it down significantly. Move that operation back into the parent process.

### CODE REVIEW

**Comment by sevko** [comment]
Needs to be tested in a dev import.

File: src/centroid.js [hunk 1]
```diff
@@ -11,11 +11,8 @@ module.exports = function( geometry, cb ){
     [ bounds[2], bounds[3] ]
   ]);
 
-  if( center.latitude && center.longitude ){
+  if( center.latitude !== undefined && center.longitude !== undefined ){
     var centroid = { lat: center.latitude, lon: center.longitude };
-
-    return cb( null, centroid );
+    return centroid;
   }
```
Comment (dianashk) [inline src/centroid.js]: Would be cleaner to return something explicit here: false, null, []? Not having a return for a function that's expected to return a value is not cool.

**Comment by sevko** [comment]
Everything looks fine in the latest dev import. A [sample query](http://pelias.dev.mapzen.com/suggest/coarse?bbox=42.61779143282346,-79.12902832031249,38.466492845389446,-84.48486328124999&input=he&lat=40.574&lon=-81.807&size=10&zoom=8) returns the following results:

```javascript
{
    "bbox": [
        -85.10653,
        33.472521,
        -0.275005,
        51.840105
    ],
    "date": 1425934658090,
    "features": [
        {
            "geometry": {
                "coordinates": [
                    -84.111525,
                    41.326702
                ],
                "type": "Point"
            },
            "properties": {
                "admin0": "United States",
                "admin1": "Ohio",
                "admin2": "Henry",
                "alpha3": "USA",
                "id": "313:adm2:us:usa:henry",
                "layer": "admin2",
                "name": "Henry",
                "text": "Henry, Ohio",
                "type": "admin2"
            },
            "type": "Feature"
        },
        {
            "geometry": {
                "coordinates": [
                    -0.275005,
                    51.840105
                ],
                "type": "Point"
            },
            "properties": {
                "admin0": "United Kingdom",
                "admin1": "Hertfordshire",
                "alpha3": "GBR",
                "id": "168:adm1:gb:gbr:hertfordshire",
                "layer": "admin1",
                "name": "Hertfordshire",
                "text": "Hertfordshire, United Kingdom",
                "type": "admin1"
            },
            "type": "Feature"
        },
        {
            "geometry": {
                "coordinates": [
                    -84.13845,
                    33.472521
                ],
                "type": "Point"
            },
            "properties": {
                "admin0": "United States",
                "admin1": "Georgia",
                "admin2": "Henry",
                "alpha3": "USA",
                "id": "1011:adm2:us:usa:henry",
                "layer": "admin2",
                "name": "Henry",
                "text": "Henry, Georgia",
                "type": "admin2"
            },
            "type": "Feature"
        },
        {
            "geometry": {
                "coordinates": [
                    -85.10653,
                    38.466212
                ],
                "type": "Point"
            },
            "properties": {
                "admin0": "United States",
                "admin1": "Kentucky",
                "admin2": "Henry",
                "alpha3": "USA",
                "id": "1589:adm2:us:usa:henry",
                "layer": "admin2",
                "name": "Henry",
                "text": "Henry, Kentucky",
                "type": "admin2"
            },
            "type": "Feature"
        },
        {
            "geometry": {
                "coordinates": [
                    -82.503221,
                    35.322422
                ],
                "type": "Point"
            },
            "properties": {
                "admin0": "United States",
                "admin1": "North Carolina",
                "admin2": "Henderson",
                "alpha3": "USA",
                "id": "159:adm2:us:usa:henderson",
                "layer": "admin2",
                "name": "Henderson",
                "text": "Henderson, North Carolina",
                "type": "admin2"
            },
            "type": "Feature"
        },
        {
            "geometry": {
                "coordinates": [
                    -84.116753,
                    39.754774
                ],
                "type": "Point"
            },
            "properties": {
                "admin0": "United States",
                "admin1": "Ohio",
                "admin2": "Montgomery",
                "id": "4888:_:_:usa:hearthstone",
                "layer": "neighborhood",
                "name": "Hearthstone",
                "text": "Hearthstone, Montgomery, Ohio",
                "type": "neighborhood"
            },
            "type": "Feature"
        },
        {
            "geometry": {
                "coordinates": [
                    -82.2052,
                    38.80975
                ],
                "type": "Point"
            },
            "properties": {
                "admin0": "United States",
                "admin1": "Ohio",
                "admin2": "Gallia",
                "id": "4901:_:_:usa:hedgewood",
                "layer": "neighborhood",
                "name": "Hedgewood",
                "text": "Hedgewood, Gallia, Ohio",
                "type": "neighborhood"
            },
            "type": "Feature"
        },
        {
            "geometry": {
                "coordinates": [
                    -81.590159,
                    40.035028
                ],
                "type": "Point"
            },
            "properties": {
                "admin0": "United States",
                "admin1": "Ohio",
                "admin2": "Guernsey",
                "id": "4917:_:_:usa:henderson_heights",
                "layer": "neighborhood",
                "name": "Henderson Heights",
                "text": "Henderson Heights, Guernsey, Ohio",
                "type": "neighborhood"
            },
            "type": "Feature"
        }
    ],
    "type": "FeatureCollection"
}
```

**Comment by dianashk** [comment]
+1

**Comment by sevko** [comment]
Waiting on @missinglink .

**Pull request merged**